lib/PromAlertProxy/Hub.pm

```diff
@@ -0,0 +1,186 @@
+package PromAlertProxy::Hub;
+
+# ABSTRACT: Proxy Prometheus alerts to other places
+
+use 5.028;
+use Moo;
+use experimental qw(signatures);
+
+use Types::Standard qw(CodeRef Object Str Int);
```

Comment:
Here you could insert my usual grumping about how I find Type::Tiny and friends super weird and hard to use and read, and (especially in a program like this, where you're gonna start it up and leave it running and compilation time is negligible as a percentage of running time) I'd just write it in Moose, which gives you most of the benefits of the other.

Reply:
The grumps have been shared and I acknowledge them.